lib/Telemetry/src/SensorSigStrength.h

```diff
@@ -0,0 +1,42 @@
+#ifndef _SENSOR_SIG_H_
+#define _SENSOR_CAN_H_
+
+#include "Particle.h"
+#include "Sensor.h"
+
+class SensorSigStrength : public Sensor
+{
+    public:
+
+        /**
+         * Constructor
+         * @param /No external dependencies
+        **/
+        SensorSigStrength();
+
+        /**
+         *Empty handle since signal strength and quality run on publish
+        **/
+        void begin();
+
+        /**
+         * Empty handle since signal strength and quality run on publish
+        **/
+        void handle();
+
+        String getHumanName();
+
+        float getStrength(); 
+        float getQuality();
+
+        /**
+         * @return Signal strength and quality in percentage
+        **/
+    
+    private:
+        int _strength;
+        int _quality;
```

Comment:
The member variables _strength and _quality aren't necessary right now as we don't need to store anything in this sensor.  Could just return value from _sig.get...()